src/app/boards/components/board-item/tasks-list/tasks-list.component.ts

```diff
@@ -0,0 +1,351 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute, Router } from '@angular/router';
+import { AuthService } from 'src/app/auth/services/auth.service';
+import { BoardsService } from 'src/app/boards/services/boards.service';
+import {
+  Board,
+  Column,
+  NewColumnData,
+  NewTaskData,
+  Task,
+} from 'src/app/models/models';
+import { forkJoin, Subscription } from 'rxjs';
+import { ModalService } from 'src/app/services/modal.service';
+import { ColumnService } from 'src/app/boards/services/column.service';
+import {
+  CdkDragDrop,
+  moveItemInArray,
+  transferArrayItem,
+} from '@angular/cdk/drag-drop';
+import { SharedService } from 'src/app/shared/shared.service';
+import { TaskService } from 'src/app/boards/services/task.service';
+
+@Component({
+  selector: 'app-tasks-list',
+  templateUrl: './tasks-list.component.html',
+  styleUrls: ['./tasks-list.component.scss'],
+})
+export class TasksListComponent implements OnInit {
+  board?: Board;
+  boardId: string | null = '';
+  columnData?: Column;
+  length: number = 0;
+  column!: Column;
+  columns: Column[] = [];
+  private newColumnSubscription: Subscription | undefined;
```

Comment:
Please try to follow a single approach to the declaration sequence. I can propose you the following

@decorated properties
public properties
private properties
constructor
lifecycle hooks
public methods
private methods